server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -61,7 +62,7 @@ case class NettyConfig(
     sslContext: Option[SslContext],
     eventLoopConfig: EventLoopConfig,
     socketConfig: NettySocketConfig,
-    initPipeline: NettyConfig => (ChannelPipeline, ChannelHandler) => Unit,
```

Comment:
the comment for initPipeline needs an update, or formalising as a trait since the ChannelHandler parameters can be confusing. Or maybe pass a list of handlers?